### DISCUSSION TITLE
Office_S_Tips/excel/2021/02/02/Excel-Hurtowa_zamiana_fragm_tekstu

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Office_S_Tips#29] [Komentarze]
# Office_S_Tips/excel/2021/02/02/Excel-Hurtowa_zamiana_fragm_tekstu

Przepis na hurtową zamianę fragmentów tekstu z uwzględnianiem wielkości znaków - z użyciem ZNAJDŹ() i PODSTAW().

https://andrzejq.github.io/Office_S_Tips/excel/2021/02/02/Excel-Hurtowa_zamiana_fragm_tekstu.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
